rest-api/dao/questionnaire_response_dao.py

```diff
@@ -241,6 +244,15 @@ def _update_participant_summary(
             new_status = QuestionnaireStatus.SUBMITTED_NO_CONSENT
           elif code.value == CONSENT_FOR_DVEHR_MODULE:
             new_status = dvehr_consent
+          elif code.value == CONSENT_FOR_STUDY_ENROLLMENT_MODULE:
+            # set language of consent to participant summary
+            for extension in resource_json.get('extension', []):
+              if extension['url'] == _LANGUAGE_EXTENSION and \
```

Comment:
You may want to use extension.get() for these, just in case there's some weird extension that doesn't contain url, or has the right url but no valueCode. (You'll get a KeyError if you use [].)